scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -96,11 +96,13 @@ <h6 class="ml-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
 
     <!-- scrollable div containing dynamic gene list -->
     <div class="mt-3 ml-3" style="max-height:200px; overflow-y: scroll; min-height: 100px;">
+      <i>
       {% for hpo_gene in case.dynamic_gene_list %}
         {{ hpo_gene.hgnc_symbol }}&nbsp;&nbsp;
       {% else %}
         <span class="text-muted">No gene list generated</span>
```

Comment:
Optionally leave this non-italic, eg by adding in a counter non-italic block. Or using an if instead of the for loop else.
```suggestion
       </i><span class="text-muted">No gene list generated</span><i>
```

Reply:
Ok I'll change it 😄

Reply:
Changed as you suggested by introducing an if condition. Tested locally and when there are no genes it looks like this:

![image](https://user-images.githubusercontent.com/28093618/140732895-2418b291-2489-4c68-a0d3-2b49232e524b.png)